feg/gateway/tools/s8_cli/main.go

```diff
@@ -521,5 +521,8 @@ func main() {
 	}
 	args := os.Args[2:]
 	cmd.Flags().Parse(args)
-	os.Exit(cmd.Handle(args))
+	// "If a test calls os.Exit, the test binary will exit, and go test will report a failure." https://github.com/golang/go/issues/45619
```

Comment:
does the issue affect s8_cli tool? The issue affects Go tests not applications. So, why this change?

~/magma/feg/gateway/tools$ go test ./...
?   	magma/feg/gateway/tools/gw_csfb_service_cli	[no test files]
?   	magma/feg/gateway/tools/gw_s6a_service_cli	[no test files]
?   	magma/feg/gateway/tools/gx_client_cli	[no test files]
?   	magma/feg/gateway/tools/gy_client_cli	[no test files]
?   	magma/feg/gateway/tools/hello_cli	[no test files]
?   	magma/feg/gateway/tools/hss_cli	[no test files]
?   	magma/feg/gateway/tools/s6a_cli	[no test files]
?   	magma/feg/gateway/tools/s8_cli	[no test files]
?   	magma/feg/gateway/tools/swx_cli	[no test files]

~/magma/feg/gateway/tools$ go version
go version go1.18 darwin/arm64

Reply:
note s8_cli has a test file that is running with 
```
go test -tags cli_test magma/feg/gateway/tools/...
```

or you can run it doing `make test`